metric/system/process/process.go

```diff
@@ -133,6 +133,8 @@ func (procStats *Stats) GetSelf() (ProcState, error) {
 	if err != nil {
 		return ProcState{}, fmt.Errorf("error fetching PID %d: %w", self, err)
 	}
+	// Copy Value over here, which would normally happen in system/process as part of percent calculation
+	pidStat.CPU.Total.Value = opt.FloatWith(float64(pidStat.CPU.Total.Ticks.ValueOr(0)))
```

Comment:
Should or could this be fixed in https://github.com/elastic/integrations/tree/main/packages/elastic_agent instead?

Reply:
Also fixing it in the integration or dashboard itself means we aren't coupled to the stack release.

Reply:
Yah, currently on the fence about how to fix this, digging into the issue a bit more, since I think there's other things at work.